sdk/trace/batch_span_processor_test.go

```diff
@@ -257,3 +257,67 @@ func TestBatchSpanProcessorShutdown(t *testing.T) {
 	}
 	assert.Equal(t, 1, bp.shutdownCount)
 }
+
+func TestBatchSpanProcessorForceFlushSucceeds(t *testing.T) {
+	te := testBatchExporter{}
+	tp := basicTracerProvider(t)
+	option := testOption{
+		name: "default BatchSpanProcessorOptions",
+		o: []sdktrace.BatchSpanProcessorOption{
+			sdktrace.WithMaxQueueSize(0),
+		},
+		wantNumSpans:   517,
+		wantBatchCount: 1,
+		genNumSpans:    517,
+	}
+	ssp := createAndRegisterBatchSP(option, &te)
+	if ssp == nil {
+		t.Fatalf("%s: Error creating new instance of BatchSpanProcessor\n", option.name)
+	}
+	tp.RegisterSpanProcessor(ssp)
+	tr := tp.Tracer("BatchSpanProcessorWithOption")
+	generateSpan(t, option.parallel, tr, option)
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		// Force flush any held span batches
+		ssp.ForceFlush(context.Background())
+	}()
+	wg.Wait()
```

Comment:
```suggestion
	// Force flush any held span batches
	ssp.ForceFlush(context.Background())
```

I don't think adding a `WaitGroup` and running `ForceFlush` in a goroutine does anything different here than running it directly.